### DISCUSSION TITLE
Welcome to wooms Discussions!

### DISCUSSION THREAD

**Discussion by aiiddqd** [wpcraft-ru/wooms#441] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by missha2022** [comment]
Не подскажите - почему сумма заказа с сайта передается в мой склад с нулевым значением?

**Comment by denisAtm** [comment]
Как метки передавать со склада?

**Comment by WEBzaytsev** [comment]
Как добавить в исключения, категорию из МС?

**Comment by aiiddqd** [comment]
уау ) я даже не знал что тут такое есть )
давайте вы зададите вопросы в своих ветках вот так 
https://github.com/wpcraft-ru/wooms/discussions/new

а тут закрываем дискотеку
